sideboard/tests/test_configuration.py

```diff
@@ -74,7 +74,7 @@ def config_overrides_set(self, monkeypatch):
     @pytest.fixture
     def plugin_dirs(self):
         module_path = '/fake/sideboard/plugins/test-plugin/test_plugin'
-        root_path = os.path.join(os.getcwd(), 'plugins', 'test-plugin')
+        root_path = os.path.join(config['plugins_dir'], 'test-plugin')
```

Comment:
Hmm, I'll need to go back and look at the git history of this change then, because my vague recollection is
- the tests weren't passing the old way
- ``config['plugins_dir']`` is supposed to be the canonical way to get the location of the plugins directory
- the tests passed when I changed it to that
- I ran the tests after the merge and they passed both locally and on Travis

So I think this is correct?  But I didn't take the time today to look at how that fixture is actually being used when making this MR, so it's possible that the tests are currently passing for the wrong reason and should be fixed so that they fail, and then further fixed so that they pass correctly.